utils/mock-data-loader/centralApi.js

```diff
@@ -49,7 +49,7 @@ const updateFacility = async (facilityId, facilityUpdate, user) => {
     url: `${DTFS_CENTRAL_API_URL}/v1/portal/facilities/${facilityId}`,
     headers,
     data: {
-      ...facilityUpdate,
+      facilityUpdate,
```

Comment:
Do we not need audit details on this? Suprised the validation isn't throwing an error